src/components/researchers/ResearchersListItem.jsx

```diff
@@ -25,77 +20,93 @@ export const ResearchersListItem = ReactTimeout(class _ResearchersListItem exten
+        'fade-in-enter': this.state.startAnimating
+      }
+    )
 
     return (
-      <div
-        className={
-          classnames(
-            'list-item',
-            'panel',
-            'slide-up',
-            'fade-in',
-            'slow',
-            {
-              'slide-up-enter': this.state.startAnimating,
-              'fade-in-enter': this.state.startAnimating
-            }
-          )
-        }
-        style={{ 'transitionDelay': `${this.props.index * 100}ms` }}
-      >
-        <div className='panel-block'>
-          <h6 className='is-size-6 has-text-weight-bold is-monospaced has-text-grey list-item--subtitle__with-margin'>
-            RESEARCHER ADDRESS
-          </h6>
-          <h5 className='title is-size-5 is-monospaced'>
-            <EtherscanAddressLink address={this.props.researcher.address}>{this.props.researcher.address}</EtherscanAddressLink>
-          </h5>
+      <div className='list-item list-item__researcher'>
+        <span
+          className={`${animatingCssClassNames}
+          list-item__cell
+          list-item__cell--id
+          has-text-grey
+          has-text-weight-light
+        `}>
+          <Link
+            to={link}
+            className='no-select title is-size-4 has-text-weight-normal'
+          >
+            #{this.props.index + 1} &nbsp;
+          </Link>
+        </span>
 
-          <br />
+        <span className={`
+          ${animatingCssClassNames}
+          list-item__cell
+          list-item__cell--total-vouched
+        `}>
+          <Link
+            to={link}
+            className='no-select'
+          >
+            <h6 className='list-item__cell--total-vouched__title is-size-6 has-text-weight-bold is-monospaced has-text-grey'>
+              TOTAL VOUCHED
+            </h6>
+            <h3 className='list-item__cell--total-vouched__value is-size-3 is-monospaced'>
+              <span className='item--version has-text-black has-text-weight-light'>
+                <ZepTokenLogo width='22' height='22' className='researcher--zep-token-icon' />{displayWeiToEther(this.props.researcher.amount)}
+              </span>
+            </h3>
+          </Link>
+        </span>
 
-          <h6 className='is-size-6 has-text-weight-bold is-monospaced has-text-grey'>
-            TOTAL VOUCHED
-          </h6>
-          <h3 className='is-size-3 is-monospaced'>
-            <span className='item--version has-text-black has-text-weight-light'>
-              <ZepTokenLogo width='22' height='22' className='researcher--zep-token-icon' />{displayWeiToEther(this.props.researcher.amount)}
+        <span className={`
+          ${animatingCssClassNames}
+          list-item__cell
+          list-item__cell--title
+        `}>
+          <Link
+            to={link}
+            className='no-select'
+          >
+            <h5 className='title is-size-5 is-monospaced has-text-link'>
+              {this.props.researcher.address}
```

Comment:
`<ResearcherNameAndAddress address={this.props.researcher.address} />`

Reply:
I didn't feel it was a good idea to show both here with a link to Etherscan. I could see us using the <EnsName /> component here, though!